src/components/ui/TemplateFormEditor.tsx

```
@@ -227,149 +225,146 @@ export default function WorkoutFormEditor({
+    setTemplateExercises(updatedExercises);
   };
 
   // Update set details (reps, weight)
   const updateSet = (
-    exerciseIndex: number, 
-    setIndex: number, 
-    field: 'reps' | 'weight', 
+    exerciseIndex: number,
+    setIndex: number,
+    field: "reps" | "weight",
     value: any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)